crates/sui-graphql-rpc/src/config.rs

```diff
@@ -12,9 +12,11 @@ use crate::functional_group::FunctionalGroup;
 // TODO: calculate proper cost limits
 const MAX_QUERY_DEPTH: u32 = 15;
 const MAX_QUERY_NODES: u32 = 50;
+const MAX_OUTPUT_NODES: u64 = 100_000; // Maximum number of output nodes allowed in the response
 const MAX_QUERY_PAYLOAD_SIZE: u32 = 2_000;
 const MAX_DB_QUERY_COST: u64 = 20_000; // Max DB query cost (normally f64) truncated
-const DEFAULT_PAGE_SIZE: u64 = 20; // Default number of elements allowed on a page of a connection
+// TODO: update this based on config
```

Comment:
I meant in the sense that DEFAULT_PAGE_SIZE remains the same while the Limits struct would have the actual desired default_page_size. But yeah this is not needed since we can grab Limits from the ctx